### DISCUSSION TITLE
Getting Properties created in a Data Package with py-capellambse #179

### DISCUSSION THREAD

**Discussion by khammoutene** [dbinfrago/py-capellambse#180] [Q&A] [answered]
Hi,

I have created in a data package, several properties (e.g. enumerations) that I'd like to access via py-capellambse.

When I "print" the data package, I cannot see anything related to the properties created in the data package.

Maybe I am missing something on how to get them?

Thank you.

**Comment by vik378** [comment, marked as answer]
@khammoutene , you may want to have a closer look at this example here: https://dsd-dbs.github.io/py-capellambse/examples/03%20Data%20Values.html

**Reply by khammoutene** [reply]
Hi @vik378 I know this example but it does not show how to get what I am looking for.

**Reply by khammoutene** [reply]
@vik378 When I look at what class.properties returns, it looks like Capella is implementing class attributes and class associations as properties: e.g; If classA has AttrA and an association AssocA to another class, classA.properties returns 2 properties: 1 for the attribute and one for the association.

You can easily see this in the Semantic Browser.

Do you know how, with py-capellambse, I can differentiate between associations and attributes?

Thank you.

**Reply by khammoutene** [reply]
Do you know how, with py-capellambse, I can differentiate between associations and attributes?
